python/cudf/requirements/cuda-10.1/dev_requirements.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+
+cupy-cuda101
```

Comment:
Do we need to include pyarrow in here? I know we can't pip install the cuda enabled version, but not sure if there's a mechanism to indicate it's needed.

Reply:
Should we add a comment in this file to install pyarrow too separately? Since this is something used by a developer they should be able to follow the url and install from source.

Reply:
I guess? I was hoping there was a syntax for "Hey this needs to exist already, otherwise fail", but I don't see anything.

Reply:
Added a comment here..